presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -481,7 +481,9 @@ void PrestoServer::run() {
       taskManager_.get(),
       memoryAllocator,
       asyncDataCache,
-      velox::connector::getAllConnectors());
+      velox::connector::getAllConnectors(),
+      this,
```

Comment:
If we provide Presto server here, then we do need to provide cache, driver executor? We can get those members from Presto server object?

Reply:
These are not exposed as public.  Do we want to expose them?